packages/database/src/server/syncServerModelClasses/Answer.js

```diff
@@ -0,0 +1,33 @@
+import { S3, S3Client } from '@tupaia/server-utils';
+import { AnswerModel as AnswerModelBase } from '../../core/modelClasses';
+
+/**
+ * Extends from base AnswerModel to handle server-side image and file sync.
+ */
+export class AnswerModel extends AnswerModelBase {
```

Comment:
Purely for consistency with other subclasses (bit more discoverable when <kbd>⌘</kbd><kbd>F</kbd>-ing around):

```suggestion
import { AnswerModel as BaseAnswerModel } from '../../core/modelClasses';

/**
 * Extends from base AnswerModel to handle server-side image and file sync.
 */
export class AnswerModel extends BaseAnswerModel {
```